stdlib/nodejs/env.rb

```diff
@@ -0,0 +1,46 @@
+ENV = Object.new
+
+def ENV.[]=(name, value)
+  `process.env[#{name.to_s}] = #{value.to_s}`
+end
+
+def ENV.[](name)
+  `process.env[#{name}] || nil`
+end
+
+def ENV.key?(name)
+  `process.env.hasOwnProperty(#{name})`
+end
+
+# alias
+def ENV.has_key?(name)
+  ENV.key? name
+end
+
+def ENV.include?(name)
+  ENV.key? name
+end
+
+def ENV.member?(name)
+  ENV.key? name
+end
+
+def ENV.empty?
+  `Object.keys(process.env).length === 0`
+end
+
+def ENV.keys
+  `Object.keys(process.env)`
+end
+
+def ENV.delete(name)
+  %x{
+    var value = process.env[#{name}] || nil;
+    delete process.env[#{name}];
+    return value;
+  }
+end
+
+def ENV.to_s
+  'ENV'
+end
```

Comment:
The whole file can be changed to use `class << ENV`, I think that would be more clean:

```rb
class << ENV
  def [](name)
    `process.env[#{name}] || nil`
  end
  # …
end
```

It also makes easier to do `alias` (e.g. for `key?` / `has_key?`)

Reply:
This is the syntax I was looking for, thanks :100:

Reply:
Done